### ISSUE TITLE
What events trigger LayoutShift Performance Callback?

### ISSUE THREAD

**Issue by Nithanaroy** [WICG/layout-instability#92]
- LS computation is skipped for a short interval on user input. Does that mean the LayoutShift algorithm always runs in the browser otherwise? 
- Are there any (heuristic) events after which LS observer cb _may_ fire?

**Comment by Nithanaroy** [comment]
Or is it only computed when we subscribe to it in JS?

**Comment by npm1** [comment]
> * LS computation is skipped for a short interval on user input. Does that mean the LayoutShift algorithm always runs in the browser otherwise?

It's not skipped, you would still get entries with `hadRecentInput` set to true.

> * Are there any (heuristic) events after which LS observer cb _may_ fire?

No, how this works is an entry is created when a shift happens. If there is no observer task queued, we queue one. And the observer callback is ran with whichever entries are available when the observer task runs. See https://w3c.github.io/performance-timeline/#the-performanceobserver-interface

> Or is it only computed when we subscribe to it in JS? My guess is not, but just wanted to confirm

No, as the buffered flag allows you to obtain shifts that occurred in the past.

Closing as I think I've addressed the questions (no issue being reported here).
